app/repositories/versioned_manual_repository.rb

```diff
@@ -0,0 +1,111 @@
+require "manual_repository"
+
+class VersionedManualRepository
+
+  class NotFoundError < StandardError; include ManualRepository::NotFoundError; end
+
+  def self.get_manual(manual_id)
+    new.get_manual(manual_id)
+  end
+
+  def get_manual(manual_id)
+    manual_record = ManualRecord.find_by(manual_id: manual_id)
+    raise NotFoundError if manual_record.nil?
+
+    {
+      draft: get_current_draft_version_of_manual(manual_record),
+      published: get_current_published_version_of_manual(manual_record),
+    }
+  end
+
```

Comment:
It looks like we can keep the public methods of this class to just the methods above this line.